e2e/bitrise.yml

```diff
@@ -48,26 +48,11 @@ workflows:
             set -ex
 
             envman add --key VERSION_CODE --value $(( $BITRISE_BUILD_NUMBER * 10 ))
-    - change-android-versioncode-and-versionname:
-        inputs:
-        - new_version_code: $VERSION_CODE
-        - version_code_offset: "0"
-        - build_gradle_path: $BITRISE_SOURCE_DIR/Application/build.gradle
     - change-android-versioncode-and-versionname:
         inputs:
         - new_version_code: $VERSION_CODE
         - version_code_offset: "1"
         - build_gradle_path: $BITRISE_SOURCE_DIR/Wearable/build.gradle
-    - android-build:
```

Comment:
Why the removal of this? AFAIK this E2E test was here to test uploading multiple AAB files.

Reply:
Will have to find an other way for that, you need to upload wearOS bundles on a seperate track:
```Changed multiple aab upload test, wearOS is now required to be on a separate track: https://support.google.com/googleplay/android-developer/answer/13295490#zippy=%2Cwhen-does-the-default-track-stop-serving-wear-os-devices```